docs/release-info/release-notes.md

```diff
@@ -220,6 +226,12 @@ You can now ingest data from multiple storage accounts using the new `azureStora
 
 [#15630](https://github.com/apache/druid/pull/15630)
 
+#### Added a new config to `AzureAccountConfig`
+
+The new config `storageAccountEndpointSuffix` lets you configure the endpoint suffix to use so that you can override the default and connect to other endpoints, such as Azure Government.
```

Comment:
```suggestion
The new config `storageAccountEndpointSuffix` lets you configure the endpoint suffix so that you can override the default and connect to other endpoints, such as Azure Government.
```